docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+## Contributing to Haveno
```

Comment:
Why is this file all uppercase and the others are lowercase?  Should be consistent.

Reply:
uppercase files are meant to interact with GitHub's UI: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors